deploy/role.yaml

```diff
@@ -0,0 +1,43 @@
+kind: Role
```

Comment:
License header.

Reply:
Is it really needed to add the license header to al YAML manifests?